### PR TITLE
Adding tool to produce single module occupancy trends

### PR DESCRIPTION
**Pull request by fioriNTU**
Adding a tool for Tracker offline shifters to produce single module occupancy trend (same as #16360 for 80X)

### CODE REVIEW

File: DQM/SiStripMonitorClient/bin/BuildFile.xml [hunk 1]
```diff
@@ -6,3 +6,4 @@
 <bin   name="ls_cert" file="ls_cert.cc"></bin>
 <bin   name="lsbs_cert" file="lsbs_cert.cc"></bin>
 <bin   name="moduleOccupancyPlots" file="moduleOccupancyPlots.cc"></bin>
+<bin   name="moduleOccupancyTrend" file="moduleOccupancyTrend.cc"></bin>
```
Comment (dmitrijus) [inline DQM/SiStripMonitorClient/bin/BuildFile.xml]: Perhaps you should include 'tracker' into the name?

In any case, you are polluting the global 'executable' path, but I do not know of any better way to put this executable. But then this is not really a test...
Perhaps you can try to put it in "test/" subfolder.

**Comment by dmitrijus** [comment]
+1

I made a small, non-critical comment about the bin/executable usage. If you want to follow up with it - please add more commits and I will sign again.

**Comment by venturia** [comment]
It is possible that this specific use case, the trend plot maker, could be handles with a script (to be put in /script) instead of an executable (to be put in /bin). But in general the comment that adding new useful and critical tools in CMSSW is "polluting the path" will not be received by the Tracker Project

**Comment by smuzaffar** [comment]
please test
there was an issue with local siteconf which should be fixed now

**Comment by davidlange6** [comment]
+1

**Pull request merged**